webrtc.html

```diff
@@ -288,6 +288,10 @@ <h2>
                     An array of objects describing servers available to be used
                     by ICE, such as STUN and TURN servers.
                   </p>
+                  <p class="note">
+                    Existing implementations only utilize the first 32 entries;
+                    the others are ignored.
+                  </p>
                 </dd>
```

Comment:
I'd leave the language as normative in the processing steps, but formulate as

If the number of ICE servers exceeds an implementation-defined limit, ignore the ICE servers above the threshold.
This implementation defined limit MUST be at least 32.